src/containers/WelcomePage/components/TableComponent.tsx

```diff
@@ -191,11 +163,13 @@ const TableComponent = <T extends string>({
         ) : null}
       </StyledTable>
       {isPending ? (
-        <SpinnerWrapper>
-          <Spinner appearance="small" />
-        </SpinnerWrapper>
+        <LoadingNoContentWrapper>
+          <Spinner />
+        </LoadingNoContentWrapper>
       ) : noResultsText && isEmpty(tableData.flat()) ? (
-        <NoResultsText>{noResultsText}</NoResultsText>
+        <LoadingNoContentWrapper>
```

Comment:
Hvorfor trenger vi en flex-wrapper her? Er det ikke bare ett element?

Reply:
var egentlig bare for å midtstille innholdet, jeg kan eventuelt gjøre midtstilling i TableWrapper istedet, så slipper vi wrapper

Reply:
var egentlig bare for å midtstille innholdet, eventuelt må det legges i `TableWrapper`

Reply:
Holder ikke textalign?

Reply:
Forsåvidt i dette tilfellet, men den brukes også som wrapper for spinneren